developers_chamber/qa/checks.py

```diff
@@ -52,7 +52,7 @@ def _run_check(self):
 
         translation_files = []
         for diff in self._get_unstaged():
-            if self._is_translation_file(diff.b_path) and re.findall(r'(\+|-)(msgstr|msgid)', diff.diff.decode()):
```

Comment:
This is no longer needed, since modification date is no longer included. But the main problem was, that this ignored changes in multiline strings - i.e. ignored any changed lines that were not beginning with `msgstr` or `msgid`. That is not desired.